openbb_platform/CONTRIBUTING.md

```diff
@@ -130,9 +130,9 @@ Usage of these models will unlock a set of perks that are only available to stan
 - Can expect consistent data types and validation.
 - Will work seamlessly with other providers that use the same standard model.
 
-The standard models are defined under the `/OpenBBTerminal/openbb_platform/platform/core/provider/openbb_core/provider/standard_models/` directory.
+The standard models are defined under the `/OpenBBTerminal/openbb_platform/core/openbb_core/provider/standard_models` directory.
```

Comment:
Since you're at it, can you rename the repo where it's mentioned to OpenBB vs the old OpenBBTerminal?